features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -185,15 +185,22 @@ class WisunInterface : public MeshInterfaceNanostack {
     /**
      * \brief Set Wi-SUN network size.
      *
-     * Function stores new parameters to mbed-mesh-api and uses them when connect() is called next time.
-     * If device is already connected to the Wi-SUN network then device will restart network discovery after
+     * Function stores network size parameter to the mbed-mesh-api and uses it when connect() is called for the next
+     * time. If device is already connected to the Wi-SUN network then device will restart network discovery after
      * changing the network size.
      *
-     * Default value: medium
-     * Small network size: less than hundred devices
-     * Medium network size: hundreds of devices
-     * Large network size: thousands of devices
-     * Certificate: used on testing
+     * It is recommended to set the correct network size because the Wi-SUN network configuration parameters are
+     * adjusted based on the selected network size. A network configured for a small amount of devices may not work
+     * optimally for large amount of devices.
+     *
+     * The network size is measured as hundreds of devices. For example:
+     * - set network size to 1 to use Wi-SUN network with less than 100 devices (small).
+     * - set network size to 8 to use Wi-SUN network with 100-800 devices (medium).
```

Comment:
We need to remove the limits on devices as the amount of devices is dependent on the data rate so
medium settings is used from 500, 1000, 1500 to 2500 devices depending the data rate 

So it would be better to just say that there is 4 buckets of settings which are selected based on data rate and device amount

This field ONLY gives the device amount 
so
1 = 100
2=200
5=500
20=2000